engine/apps/metrics_exporter/tasks.py

```diff
@@ -40,51 +45,62 @@ def save_organizations_ids_in_cache():
 def start_calculate_and_cache_metrics(metrics_to_recalculate: list[RecalculateOrgMetricsDict]):
     """Start calculation metrics for each object in metrics_to_recalculate"""
     for counter, recalculation_data in enumerate(metrics_to_recalculate):
+        if not is_allowed_to_start_metrics_calculation(**recalculation_data):
+            continue
         # start immediately if recalculation starting has been forced
         countdown = 0 if recalculation_data.get("force") else counter
         calculate_and_cache_metrics.apply_async(kwargs=recalculation_data, countdown=countdown)
+        calculate_and_cache_user_was_notified_metric.apply_async(
+            (recalculation_data["organization_id"],), countdown=countdown
+        )
+
+
+@shared_dedicated_queue_retry_task(autoretry_for=(Exception,), retry_backoff=True, max_retries=0)
+def start_recalculation_for_new_metric(metric_name):
```

Comment:
I think you are missing to add this new task to the `CELERY_TASK_ROUTES` setting